packages/grid/x-data-grid/src/components/menu/columnMenu/default/GridColumnMenuDefault.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Divider from '@mui/material/Divider';
+import { useGridColumnMenuComponents } from '../../../../hooks/features/columnMenu/useGridColumnMenuComponents';
+import { GridColumnMenuDefaultContainer } from './GridColumnMenuDefaultContainer';
+import { GridColumnMenuProps } from '../GridColumnMenuProps';
+import { GridColumnMenuColumnsItem } from './GridColumnMenuColumnsItem';
+import { GridColumnMenuFilterItem } from './GridColumnMenuFilterItem';
+import { GridColumnMenuHideItem } from './GridColumnMenuHideItem';
+import { GridColumnMenuSortItem } from './GridColumnMenuSortItem';
+import { useGridPrivateApiContext } from '../../../../hooks/utils/useGridPrivateApiContext';
+
+export const GRID_COLUMN_MENU_DEFAULT_COMPONENTS = {
+  ColumnMenuSortItem: GridColumnMenuSortItem,
+  ColumnMenuFilterItem: GridColumnMenuFilterItem,
+  ColumnMenuHideItem: GridColumnMenuHideItem,
+  ColumnMenuColumnsItem: GridColumnMenuColumnsItem,
+};
+
+export const GRID_COLUMN_MENU_DEFAULT_COMPONENTS_PROPS = {
+  columnMenuSortItem: { displayOrder: 0 },
+  columnMenuFilterItem: { displayOrder: 10 },
+  columnMenuHideItem: { displayOrder: 20 },
+  columnMenuColumnsItem: { displayOrder: 30 },
+};
+
+const GridColumnMenuDefault = React.forwardRef<HTMLUListElement, GridColumnMenuProps>(
```

Comment:
That's something I thought a lot about too, I ended up going with `Default` and `Simple` to both follow the same convention, and also this way existing users will be aware that the design got revamped so that they can migrate accordingly (if they still want `Simple`, go for it or upgrade to `Default`). But the point of removing the simple variant is also reasonable.

Reply:
In that case may be let's keep them named `GridColumnMenuDefault` and `GridColumnMenuSimple` for above mentioned reasons?
In v7 we can always do some breaking changes, and who knows users keep liking the simple one and we decide to keep it or add more variants, keeping in mind we have [pivoting](https://github.com/mui/mui-x/issues/214) in pipeline and the idea is to [integrate it in columns panel](https://github.com/mui/mui-x/issues/5700) or may be in a new variant of column menu?